### PR TITLE
Fix #1295: Add/Edit Profile Picture Button Not Well Displayed

### LINKED ISSUE #1295: This theme's (Tan with no colors) button is not well displayed.
librehealth_ehr

### PR DESCRIPTION
**Pull request by jonathanzhang53**
# Fixes #1295 

## Changes
Added `<span></span>` to button so that the other side of the button would show up according to the css files. Deleted `btn` class and added `iframe` class for consistency with the 'Delete' button's classes that's right beside it on the page.

Also deleted unnecessary end tags `</b></form>`

## Screenshot:
image

### CODE REVIEW

File: interface/patient_file/summary/demographics.php [hunk 1]
```diff
@@ -520,16 +520,16 @@ function setMyPatient() {
         "</span></td>";
       if ($arr['picture_url']) {
           echo '<td style="padding-left:1em;"><input type="file" name="profile_picture" id="files" onchange="this.form.submit()" class="hidden" style="display:none;"/>
-          <label for="files" class="btn css_button cp-positive" id="file_input_button"><i class="fa fa-pencil"></i> ';
+          <label for="files" class="iframe css_button cp-positive" id="file_input_button"><span><i class="fa fa-pencil"></i> ';
           echo xlt('Edit Profile Picture');
-          echo '</label></b></form>
```
Comment (robbyoconnor) [inline interface/patient_file/summary/demographics.php]: Why is the is the `</form>` tag removed

**Comment by Trodrige** [comment]
This looks good

File: interface/patient_file/summary/demographics.php [hunk 2]
```diff
@@ -520,16 +520,16 @@ function setMyPatient() {
...
+          echo '</span></label>
                 </td>';
       }
       else {
            echo '<td style="padding-left:1em;"><input type="file" name="profile_picture" id="files" onchange="this.form.submit()" class="hidden" style="display:none;"/>
-          <label for="files" class="btn css_button cp-positive" id="file_input_button"><i class="fa fa-plus"></i> ';
+          <label for="files" class="iframe css_button cp-positive" id="file_input_button"><span><i class="fa fa-plus"></i> ';
           echo xlt('Add Profile Picture');
-          echo '</label></b></form>
```
Comment (robbyoconnor) [inline interface/patient_file/summary/demographics.php]: Why is the is the `</form>` tag removed

File: interface/patient_file/summary/demographics.php [hunk 3]
```diff
@@ -520,16 +520,16 @@ function setMyPatient() {
         "</span></td>";
       if ($arr['picture_url']) {
           echo '<td style="padding-left:1em;"><input type="file" name="profile_picture" id="files" onchange="this.form.submit()" class="hidden" style="display:none;"/>
-          <label for="files" class="btn css_button cp-positive" id="file_input_button"><i class="fa fa-pencil"></i> ';
+          <label for="files" class="iframe css_button cp-positive" id="file_input_button"><span><i class="fa fa-pencil"></i> ';
```
Comment (muarachmann) [inline interface/patient_file/summary/demographics.php]: Tested works fine 👍

**Comment by aethelwulffe** [comment]
Merging this, even though there is a "move pictures" site change.  This should not affect that.  Pretty sure I have checked the code cross-wise, but I need to test a merged version, not just this branch pasted into the other branch.

**Pull request merged**